oak/server/rust/oak_abi/src/label/mod.rs

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Labels are defined at the ABI level because they need to be available:
+//!
+//! - to applications (e.g. via the SDK), so that they can be manipulated by running Oak Nodes
+//! - to the Runtime, so that they can be checked and enforced when data is exchanged between Nodes
+//!   over Channels
+//!
+//! In order to support this, Labels are serialized and deserialized across the ABI boundary, and
+//! can be read and write using ABI functions.
```

Comment:
(pre-existing typo) written

Reply:
Already fixed in a later commit (an inevitable downside of reviewing commit-by-commit).